boards/phytec/phycore_rt1170/Kconfig.defconfig

```diff
@@ -0,0 +1,56 @@
+# PHYCORE-RT1170
+
+# Copyright 2024, PHYTEC America, LLC
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_PHYCORE_RT1170
+
+# Only use DCD when booting primary core (M7)
+config DEVICE_CONFIGURATION_DATA
+	default y if CPU_CORTEX_M7
+
+config NXP_IMX_EXTERNAL_SDRAM
+	default y if CPU_CORTEX_M7
+
+if SECOND_CORE_MCUX && CPU_CORTEX_M4
+
+config BUILD_OUTPUT_INFO_HEADER
+	default y
+
+DT_CHOSEN_IMAGE_M4 = nxp,m4-partition
+
+# Adjust the offset of the output image if building for RT11xx SOC
+config BUILD_OUTPUT_ADJUST_LMA
+	default "($(dt_chosen_reg_addr_hex,$(DT_CHOSEN_IMAGE_M4)) + \
+	$(dt_node_reg_addr_hex,/soc/spi@400cc000,1)) - \
+	$(dt_node_reg_addr_hex,/soc/ocram@20200000)"
+
+endif
+
+if DISK_DRIVERS
```

Comment:
This block can be dropped, because this board does not have USDHC support enabled